x-pack/plugins/apm/public/components/app/Main/routeConfig.tsx

```diff
@@ -43,18 +30,57 @@ const renderAsRedirectTo = (to: string) => {
   );
 };
 
-export const routes: Route[] = [
+export const routes: BreadcrumbRoute[] = [
   {
     exact: true,
     path: '/',
     render: renderAsRedirectTo('/services'),
     breadcrumb: 'APM'
   },
+  {
```

Comment:
If we wrap all of our main routes in one <Switch>, we don't need to worry about the "switchRoutes" logic and can simplify the routes config here, and no longer need to "flatten" the routes before giving the routes config to the breadcrumbs component, etc.